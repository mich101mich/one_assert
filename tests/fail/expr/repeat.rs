fn main() {
    one_assert::assert!([false; 3]);
    one_assert::assert!([1; 3]);
}
