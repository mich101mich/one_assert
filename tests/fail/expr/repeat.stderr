error[E0308]: mismatched types
 --> tests/fail/expr/repeat.rs:2:25
  |
2 |     one_assert::assert!([false; 3]);
  |                         ^^^^^^^^^^ expected `bool`, found `[bool; 3]`

error[E0308]: mismatched types
 --> tests/fail/expr/repeat.rs:3:25
  |
3 |     one_assert::assert!([1; 3]);
  |                         ^^^^^^ expected `bool`, found `[{integer}; 3]`
